docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -140,20 +200,27 @@ _sql_db_isclean(){
 _sql_write_agentid(){
     # Auxiliary function to write the current agent build id into the sql database
     echo "$FUNCNAME: Preserving the current WMA_BUILD_ID and HostName at database: $wmaDBName."
+    local createCmd="create table wma_init(init_param varchar(100) not null unique, init_value varchar(100) not null);"
     case $AGENT_FLAVOR in
         'oracle')
-            echo "Not implemented"
+            echo "$FUNCNAME: Creating wma_init table at database: $wmaDBName"
+            _exec_oracle "$createCmd" || return
+
+            echo "$FUNCNAME: Inserting current Agent's build id and hostname at database: $wmaDBName"
+            _exec_oracle "insert into wma_init (init_param, init_value) values ('wma_build_id', '$WMA_BUILD_ID');" || return
+            _exec_oracle "insert into wma_init (init_param, init_value) values ('wma_tag', '$WMA_TAG');"           || return
+            _exec_oracle "insert into wma_init (init_param, init_value) values ('hostname', '$HOSTNAME');"         || return
+            _exec_oracle "insert into wma_init (init_param, init_value) values ('is_active', 'true');"             || return
             ;;
         'mysql')
-            local sqlCmd=""
-
             echo "$FUNCNAME: Creating wma_init table at database: $wmaDBName"
-            sqlCmd="create table wma_init(init_param varchar(100) not null, init_value varchar(100));"
-            _exec_mysql "$sqlCmd" $wmaDBName || return
+            _exec_mysql "$createCmd" $wmaDBName || return
 
             echo "$FUNCNAME: Inserting current Agent's build id and hostname at database: $wmaDBName"
-            sqlCmd="insert into wma_init (init_param, init_value) values ('wma_build_id', '$WMA_BUILD_ID'), ('hostname', '$HOSTNAME'), ('is_active', 'true');"
-            _exec_mysql "$sqlCmd" $wmaDBName || return
+            _exec_oracle "insert into wma_init (init_param, init_value) values ('wma_build_id', '$WMA_BUILD_ID');" $wmaDBName || return
```

Comment:
Shouldn't these 4 lines below actually execute `_exec_mysql` instead of `_exec_oracle`?

Reply:
A good example of why when you work with one feature you should not mess up with refactoring another one. Hence my decision above not to change the `db-prompt` call for MariaDB to refer to `_exec_mysql` with the current PR, but left it for the future. Here in this case I wanted to align the calls for the two databases to be implemented the same way. 

I'll fix that  with a short PR only for those  4 lines.